apps/pie-docs/src/components/icon-button/overview/sizes.json

```diff
@@ -0,0 +1,34 @@
+{
+  "headings": [
+    "Size",
+    "Icon size",
+    "Container size",
+    "Notes"
+  ],
+  "rows": [
+    [
+      "Large",
+      "28px",
+      "56px",
+      "-"
+    ],
+    [
+      "Medium",
+      "24px",
+      "48px",
+      "Medium is the default size of the component and should be used whenever possible."
```

Comment:
Personally I think this table would look better if this was outside the table and we removed the mostly-empty column.

Reply:
I also wonder if "Button height" might be easier to understand than "Container size"

Reply:
Probably a question @nazanin when's she back 💯